Jenkinsfile

```diff
@@ -29,7 +29,6 @@ node(label: "jenkins-slave") {
   finally {
     stage("Clean") {
       cleanWs()
-      sh 'git clean -dx --force'
```

Comment:
mmmh

Reply:
It's quite effective, although a bit harsh. I might bring back what was there before as it didn't change the outcome of tests.